nginx.conf.template

```diff
@@ -144,9 +144,25 @@ http {
         body_filter_by_lua  'ngx.ctx.buffered = (ngx.ctx.buffered or "") .. string.sub(ngx.arg[1], 1, 1000)
                             if ngx.arg[2] then ngx.var.resp_body = ngx.ctx.buffered end';
         header_filter_by_lua 'ngx.var.resp_headers = cjson.encode(ngx.resp.get_headers())';
+        
+        add_header 'Access-Control-Allow-Origin' '*';
```

Comment:
As far as I remember some browsers complain if the allowed origin is `*`, since they want the exact name, not a wildcard. But we can try to deploy it somewhere and test it.